lib/idx/types/index.ts

```diff
@@ -38,8 +38,11 @@ export enum AuthenticatorKey {
   GOOGLE_AUTHENTICATOR = 'google_otp',
 }
 
-type Input = {
+export type Input = {
```

Comment:
Not sure if there was any specific reason we did not want to export the type or if it just hadn't been used externally before.